src/Containers/ForecastPage/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+import { Box } from "@mui/material";
```

Comment:
Fix imports for all files this should be 
`import Box from "@mui/material/Box";`

Reply:
Done